### DISCUSSION TITLE
nodejs es6 import

### DISCUSSION THREAD

**Discussion by bitdom8** [ivanhofer/typesafe-i18n#353] [Q&A] [answered]
Great repo thanks. However one issue: Using nodejs ES6 module as

import LL from './src/i18n/i18n-svelte.js';

not working. As you know NodeJS works with require however me is using import

`ReferenceError: exports is not defined in ES module scope`

**Comment by bitdom8** [comment]
Also, using sveltekit as fullstack thus need to use svelte and node adapter (ES6 2015) at the same time. Any suggestions. 

Edit: Actually noticed that I can't import it for my JS server file stating that ".ts" is not recognized. How to integrate? Nodejs is very challanging with all puppeteer and other modules with .ts (typescript) support. Thanks

**Comment by ivanhofer** [comment, marked as answer]
Have you tried the [`esmImports` option](https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/generator#esmImports)?

If you need to use two different adapters, you would need to generate the code twice. Set the adapter to an option and run the generator. Then do the same again for the other adapter.

You could run your node code together with TypeScript with a tool like [`tsx`](https://github.com/esbuild-kit/tsx).

**Comment by bitdom8** [comment]
Thanks @ivanhofer . That has resolved my inquiry. The best library for i18n and great guider you are
